tools/bqtag/README.md

```diff
@@ -0,0 +1,374 @@
+            ]
+        },
+        "type": "STRING"
+        }
+    ],
+    "mode": "NULLABLE",
+    "name": "address",
+    "type": "RECORD"
+    }
+]
+```
+
+This function returns the schema of the table created as a dictionary. If there was an error or exception, a `{}` is returned. 
+
+## Create an Authorized View using Tags
+
+`create_view()` function can create the authorized BQ view by only including columns that are tagged with a particular value. While creating the view, this function takes care of the format of original table structure meaning if original table has nested and repeated columns then the created view would also have nested and repeated columns. Dataset and BigQuery Project, defined at the time of initialization of the object is used as the dataset and project for the new view. This function takes the following attributes:
+
+`table_name` is the name of the source table on which the view would be based.
+
+`view_name` is the name of the new view to be created.
+
+`tags` is the list of the tags which are to be included in the view.
+
+This function return the SQL query which is used to create the view. If there is an error or exception then an empty string is returned.
+
+```python
+VIEW1 = "view_medium"
+VIEW1_TAGS = ["medium", "low"]
+bqtv.create_view(TABLE_TO_CREATE, VIEW1, VIEW1_TAGS)
+```
+
+This would result in a new view named "view_medium" and the schema of the new view would be:
+
+```json
+[
+    {
+    "mode": "NULLABLE",
+    "name": "customer_name",
+    "type": "STRING"
+    },
+    {
+    "mode": "NULLABLE",
+    "name": "email_address",
+    "type": "STRING"
+    },
+    {
+    "fields": [
+        {
+        "mode": "NULLABLE",
+        "name": "pincode",
+        "type": "STRING"
+        },
+        {
+        "mode": "NULLABLE",
+        "name": "city",
+        "type": "STRING"
+        },
+        {
+        "mode": "NULLABLE",
+        "name": "state",
+        "type": "STRING"
+        }
+    ],
+    "mode": "NULLABLE",
+    "name": "address",
+    "type": "RECORD"
+    },
+    {
+    "mode": "NULLABLE",
+    "name": "customer_id",
+    "type": "STRING"
+    }
+]
+```
+
+# Installation
+
+## Requirements
+
+- You'll need to [download Python 3.6 or later](https://www.python.org/downloads/)
+- You'll need to [download latest version on pip](https://pip.pypa.io/en/stable/installing/)
+
+## Installing BQTag
+
+```
+git clone https://github.com/GoogleCloudPlatform/professional-services.git
+cd professional-services/tools/bqtag
+python3 -m pip install .
+```
+
+### Optional: Install in virtualenv
+
+```
+python3 -m pip install --user virtualenv
+python3 -m virtualenv venv
+source venv/bin/activate
+git clone https://github.com/GoogleCloudPlatform/professional-services.git
+cd professional-services/tools/bqtag
+python3 -m pip install .
+```
+
+## Disclaimer
+
+This is not an official Google project.
```

Comment:
I would say "This is not an official Google product". It is in a way a Google project, since it is published in a repo in the Google organization.

Reply:
Included the suggestion

Reply:
Changed to

This is not an official Google product.